vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -83,6 +84,8 @@ packages:
          - firewalld
          - python3-six
          - httpd-tools
+        pip3:
+         - jmespath
```

Comment:
There is no Ansbile task for this list. That needs to be added.

Reply:
@fmuyassarov task added, please review that I got it right.